data_steward/resources/achilles_dml.sql

```diff
@@ -2262,9 +1703,9 @@ insert into synpuf_100.achilles_results (analysis_id, stratum_1, stratum_2, coun
 --
 -- 903	Number of distinct drug era concepts per person
 insert into synpuf_100.achilles_results_dist (analysis_id, count_value, min_value, max_value, avg_value, stdev_value, median_value, p10_value, p25_value, p75_value, p90_value)
-WITH rawdata as ( select  COUNT(distinct de1.drug_concept_id)  as count_value   from  synpuf_100.drug_era de1
+WITH rawdata as ( select  cast(COUNT(distinct de1.drug_concept_id) as float64) as count_value   from  synpuf_100.drug_era de1
 	 group by  de1.person_id
- ), overallstats  as (select  cast(avg(1.0 * count_value)  as float64)  as avg_value, cast(STDDEV(count_value)  as float64)  as stdev_value, min(count_value)  as min_value, max(count_value)  as max_value, COUNT(*)  as total  from  rawdata
```

Comment:
This seems more significant than some of the other changes, is this because `avg_value` is an integer in the schema?

Reply:
`avg_value` would be the same as `count_value`, and we wanted it to be a float since that was the column type. However, `count_value` was an integer previously, requiring the cast to float for `avg_value`. It also required casting for `stdev_value`, `min_value`, `max_value` (they should've been casted too, but BQ insert is lenient in this regard). I cast `count_value` as a float so we don't have to cast for the derivatives.